### DISCUSSION TITLE
Projects의 Field 값들은 어떤 것들이 필요할까요??

### DISCUSSION THREAD

**Discussion by ms1011** [OmokNoonE/PPM-backend#7] [Ideas]
![image](https://github.com/OmokNoonE/PPM-backend/assets/64572078/14ac2c14-bb46-4cb8-b8fd-cc1e55606681)
이런 형식으로 필드를 추가할 수 있는데 어떤 값들이 필요할까요??

저는 필요하다고 생각하는 **필드**에는
- 시작일
- 종료일
- 상태

필드는 이렇게 있고 **상태 값**에는 아래와 같이 생각중입니다.
- 할 일
- 개발
- 테스트
- 배포
- 문서 작업
- 완료

자유롭게 의견 남겨주세요

**Comment by EI-FIE** [comment]
동의합니다 ~!!

**Comment by jihyeongeun** [comment]
좋은 것 같습니다!

**Comment by CJC0512** [comment]
동의합니다!
추가적으로 wiki에 특정 상태값에서 꼭 수행 해야 될  작업도 같이 정리해두면 좋을 것 같습니다.
예)
테스트
- [ ] 테스트 계획서 작성
- [ ] 기능 테스트 완료
- [ ] 결과 보고서 작성 
- [ ] 등등....

※ 단순 예시입니다.

**Reply by ms1011** [reply]
넵 좋습니다 의견 취합해서 wiki 작성할게용~

**Comment by jlee38266** [comment]
좋습니다!

**Comment by ms1011** [comment]
우선순위가 있으면 좋을 것 같습니다

**Reply by ms1011** [reply]
배포는 빼도 될 것 같습니다
